Documentation/RenderingDocs/Index.rst

```diff
@@ -10,97 +10,116 @@
 .. _render-documenation-with-docker:
 .. _render-documentation-with-docker:
 
+=========================================
+Render documentation with the TYPO3 theme
+=========================================
 
-===========================
-How to render documentation
-===========================
+..  contents::
 
-This page explains how to render a manual locally on your machine
-with Docker in order to test the rendering.
+Rendering the :file:`Documentation` folder locally with Docker
+==============================================================
 
-Run these commands in a terminal in the parent directory of the directory
-:file:`Documentation`. You should use a bash compatible shell, if possible.
+You can render the documentation of an official TYPO3 manual or a third party
+manual with the following steps:
 
-If you run into a problem while rendering, check :ref:`rendering-docs-troubleshooting`,
-`report an issue <https://github.com/t3docs/docker-render-documentation/issues/new>`__
-or :ref:`ask for help <get-help-on-writing-docs>`.
+#.  Clone the repository containing the documentation.
 
+#.  Navigate to the directory, which contains the :file:`composer.json`
 
-.. rst-class:: bignums-xxl
+#.  Check if there is documentation to be rendered:
 
-    #.  Install Docker: https://docs.docker.com/install/
+    A folder called :file:`Documentation` containing at least the files
+    :file:`Index.rst` and :file:`guides.xml`.
 
-    #.  Preparations
+#.  Choose your prefered method of rendering (See bellow):
 
-        The docker image is not listed on Docker Hub (hub.docker.com) anymore, therefore some preparations
-        need to be done once:
+Make sure that `Docker <https://www.docker.com/>`__ is installed on your system.
 
-        ..  code-block:: bash
+.. tabs::
 
-            # pull 'latest' version from GitHub container repository
-            docker pull ghcr.io/t3docs/render-documentation
+    ..  group-tab:: Linux
 
-            # The "real" tag is independent of the container repository,
-            # so let's just create that extra "real" and "generic" tag
-            docker tag ghcr.io/t3docs/render-documentation t3docs/render-documentation
+        ..  code-block:: bash
 
-            # verify it worked
-            docker run --rm t3docs/render-documentation --version
+            mkdir -p Documentation-GENERATED-temp
```

Comment:
Why one has to create the folder manually? I would expect that it is created automatically?

Reply:
Right now it is not created automatically, once we have fixed the container so that it works without we can remove this line. Right now there is an error if the directory is not present